arrow-libs/core/arrow-core-data/src/main/kotlin/arrow/core/Validated.kt

```diff
@@ -791,6 +792,7 @@ sealed class Validated<out E, out A> : ValidatedOf<E, A> {
   inline fun <B> traverse(fa: (A) -> Iterable<B>): List<Validated<E, B>> =
     fold({ emptyList() }, { a -> fa(a).map { Valid(it) } })
 
+  @Deprecated(TraverseDeprecation)
```

Comment:
Idem for all functions in this file: https://github.com/arrow-kt/arrow-core/blob/0.11.0/arrow-core-data/src/main/kotlin/arrow/core/Validated.kt